<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ATMECSINLT-060" failures="0" tests="3" name="com.atmecs.qa.falcondashboard.testscript.TC13_PassedTestCases" time="199.163" errors="1" timestamp="20 Mar 2020 14:00:18 GMT" skipped="0">
  <testcase classname="com.atmecs.qa.falcondashboard.testscript.TC13_PassedTestCases" name="dropdownPassedTestCases" time="60.640"/>
  <testcase classname="com.atmecs.qa.falcondashboard.testscript.TC13_PassedTestCases" name="dropdownPassedTestCases" time="64.680"/>
  <testcase classname="com.atmecs.qa.falcondashboard.testscript.TC13_PassedTestCases" name="dropdownPassedTestCases" time="73.843">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.atmecs.qa.falcondashboard.utils.Pageactions.getText(Pageactions.java:96)
at com.atmecs.qa.falcondashboard.helper.ValidationHelper.getDataofRow(ValidationHelper.java:27)
at com.atmecs.qa.falcondashboard.validationresults.DropdownValidation.validatePassStatus(DropdownValidation.java:34)
at com.atmecs.qa.falcondashboard.testscript.TC13_PassedTestCases.dropdownPassedTestCases(TC13_PassedTestCases.java:62)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- dropdownPassedTestCases -->
</testsuite> <!-- com.atmecs.qa.falcondashboard.testscript.TC13_PassedTestCases -->
